functions/taskrouterListeners/transfersListener.private.ts

```diff
@@ -41,16 +48,29 @@ const isChatTransfer = (
 const isChatTransferAccepted = (
   eventType: EventType,
   taskChannelUniqueName: string,
-  taskAttributes: { transferMeta?: TransferMeta },
-) => eventType === RESERVATION_ACCEPTED && isChatTransfer(taskChannelUniqueName, taskAttributes);
+  taskAttributes: ChatTransferTaskAttributes,
+) =>
+  eventType === RESERVATION_ACCEPTED &&
+  isChatTransfer(taskChannelUniqueName, taskAttributes) &&
+  taskAttributes.transferTargetType === 'worker';
 
 const isChatTransferRejected = (
   eventType: EventType,
   taskChannelUniqueName: string,
-  taskAttributes: { transferMeta?: TransferMeta },
+  taskAttributes: ChatTransferTaskAttributes,
 ) =>
   (eventType === RESERVATION_REJECTED || eventType === RESERVATION_TIMEOUT) &&
-  isChatTransfer(taskChannelUniqueName, taskAttributes);
+  isChatTransfer(taskChannelUniqueName, taskAttributes) &&
```

Comment:
Won't this assume that `taskAttributes.transferMeta.transferStatus === 'accepted'`?

Reply:
So apparently in flex, we start all our cold transfers with a status of 'accepted', which is causing me some headaches for determining whether to show the transfer buttons or not... 

We could start using the 'transferring' state for cold transfers, but then we would have to remove the check here. Could that have unexpected ramifications?